sunpy/data/sample.py

```diff
@@ -3,19 +3,16 @@
 from __future__ import absolute_import
 
 import sys
-from ._sample import sample_files as _sample_files
-import os.path
-
-from sunpy import config as _config
-_sampledata_dir = _config.get("downloads", "sample_dir")
+from ._sample import files as _sample_files
+from ._sample import base_urls as _base_urls
+from ._sample import get_sample_file
```

Comment:
what? I'm not saying anything about `get_sample_file` being something else...
In `_sample.py` file, the `_sample_files` and `_base_urls` were changed to `files` and `base_urls` for then being renamed here. I'm just suggesting to keep these variables with the original names and you don't need to rename them here.